core/datastore/src/main/java/com/droidknights/app2023/core/datastore/datasource/DefaultSettingsPreferencesDataSource.kt

```diff
@@ -9,20 +9,20 @@ import kotlinx.coroutines.flow.map
 import javax.inject.Inject
 import javax.inject.Named
 
-class SettingsPreferencesDataSource @Inject constructor(
-    @Named("setting") private val dataStore: DataStore<Preferences>
-) {
+class DefaultSettingsPreferencesDataSource @Inject constructor(
```

Comment:
datastore 하위의 class들도 internal 키워드를 활용할 수 있을 것 같아요.
```suggestion
internal class DefaultSettingsPreferencesDataSource @Inject constructor(
```

Reply:
internal  적용하였습니다.